Dockerfile

```diff
@@ -1,31 +1,32 @@
-# --------------------------------------------------------------------------
-# This is a Dockerfile to build an Ubuntu 14.04 Docker image with
-# pymssql and FreeTDS
-#
-# Use a command like:
-#     docker build -t pymssql/pymssql .
-# --------------------------------------------------------------------------
+FROM  ubuntu:18.04
+MAINTAINER  Tyler French tylerfrench2@gmail.com (@frenchtoasters)
 
-FROM  orchardup/python:2.7
-MAINTAINER  Marc Abramowitz <marc@marc-abramowitz.com> (@msabramo)
+# Install dependices
+RUN apt-get update && apt-get install -y \
+    unixodbc \
```

Comment:
@frenchtoasters why all the odbc stuff?  I believe pymssql should only need freetds and not any of the odbc libraries.